Shared/Helpers/CheckSignificantAmounts.swift

```diff
@@ -9,21 +9,31 @@ import Foundation
 
 func checkSignificantAmounts(for widgetData: TiltifyWidgetData) {
     //check if cached amount is significant
-    let cachedTotal = round((UserDefaults.shared.object(forKey: "cachedTotalRaised") as? Double) ?? -1.0)
-    let totalRaisedRaw = widgetData.totalRaised ?? -1.0
-    let totalRaised = round(totalRaisedRaw)
+    //uncomment below for testing
+//    UserDefaults.shared.set(49990.0, forKey: "cachedTotalRaised")
+    let cachedTotal = (UserDefaults.shared.object(forKey: "cachedTotalRaised") as? Double) ?? -1.0
```

Comment:
`UserDefaults.shared.double` might be better?

Reply:
i forgot to switch this as well